script/main.py

```diff
@@ -1,46 +1,52 @@
 import argparse
+
 from selenium import webdriver
-from collections import namedtuple
 from selenium.webdriver.common.by import By
-from selenium.webdriver.support.ui import WebDriverWait
 from selenium.webdriver.support import expected_conditions as EC
-
-LinkInfo = namedtuple('LinkInfo', 'name href')
+from selenium.webdriver.support.ui import WebDriverWait
 
 
 def fetch_argument():
     parser = argparse.ArgumentParser()
     parser.add_argument('--driver', help='entry driver')
-    parser.add_argument("url", help="entry page url")
+    parser.add_argument("url", help="entry page url, like: https://www.google.com")
+    parser.add_argument('--depth', type=int, help='entry how long to follow page links')
     args = parser.parse_args()
-    return args.driver, args.url
+    return args.driver, args.url, args.depth
 
 
 def create_driver(name):
     if name == 'chrome':
-        driver = webdriver.Chrome(executable_path="bin/chromedriver")
+        driver = webdriver.Chrome(executable_path="bin/chromedriver.exe")
     else:
         driver = webdriver.Firefox(executable_path="bin/geckodriver.exe")
     return driver
 
 
 def find_links(driver):
     links_elem = driver.find_elements_by_tag_name("a")
-    return [LinkInfo(elem.text, elem.get_attribute('href')) for elem in links_elem]
+    return [elem.get_attribute('href') for elem in links_elem]
 
 
-def print_link_info(links):
-    for value in links:
-        print(value.name, value.href)
+def print_urls_info(urls):
+    for item in urls:
+        print(item)
 
 
 def main():
-    driver, url = fetch_argument()
+    driver, url, depth = fetch_argument()
     driver = create_driver(driver)
-    driver.get(url)
-    WebDriverWait(driver, 10).until(EC.visibility_of_all_elements_located(By.TAG_NAME, "a"))
-    links = find_links(driver)
-    print_link_info(links)
+    urls = {url}
+    for elem in range(depth + 1):
+        new_urls = set()
+        for url in urls:
+            driver.get(url)
+            WebDriverWait(driver, 10).until(EC.presence_of_element_located((By.TAG_NAME, 'a')))
+            links = find_links(driver)
+            for item in links:
+                new_urls.add(item)
+        urls = new_urls
+    print_urls_info(urls)
```

Comment:
Please notice that you don't print all links. Only from the last page.
You should print all unique links